contrib/mesos/pkg/minion/server.go

```diff
@@ -164,7 +164,7 @@ func (ms *MinionServer) launchProxyServer() {
 // executor doesn't support failover right now, the right thing to do is to fail completely since all
 // pods will be lost upon restart and we want mesos to recover the resources from them.
 func (ms *MinionServer) launchExecutorServer(containerID string) <-chan struct{} {
-	allArgs := os.Args[1:]
+	allArgs := os.Args[2:]
```

Comment:
@jdef, looks like there is still a case where arg parsing would break: if --executor-path flag set on km scheduler:
https://github.com/kubernetes/kubernetes/blob/679e27c157637290075e65a45dfc0fde0a5c32d9/contrib/mesos/pkg/scheduler/service/service.go#L385